ProjFS.Mac/PrjFSKext/public/PrjFSHealthData.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+struct PrjFSHealthData
```

Comment:
Right now we only have vnode cache data.  Do we expect to expand it to other data?  If not we may want to make the name more specific.

Reply:
> Do we expect to expand it to other data? If not we may want to make the name more specific.

I went with the more generic name because I thought we might expand it in the future.  I'm not aware of any plans to add more stats in the near future though so I can make this more specific.